<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlumnos">
    <property name="title">
     <string>Alumnos</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAlta_alumno_2"/>
    <addaction name="actionMatricular_alumno_2"/>
    <addaction name="separator"/>
    <addaction name="actionSalir_3"/>
   </widget>
   <widget class="QMenu" name="menuMatriculas">
    <property name="title">
     <string>Matriculas</string>
    </property>
    <addaction name="actionMatricular_alumno"/>
    <addaction name="actionDesmatricular_alumno"/>
   </widget>
   <widget class="QMenu" name="menuCalificaciones">
    <property name="title">
     <string>Calificaciones</string>
    </property>
    <addaction name="actionAprobar_alumno"/>
   </widget>
   <widget class="QMenu" name="menuProfesores">
    <property name="title">
     <string>Profesores</string>
    </property>
    <addaction name="actionFicha_profesor"/>
   </widget>
   <widget class="QMenu" name="menuAyudas">
    <property name="title">
     <string>Ayudas</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuAlumnos"/>
   <addaction name="menuMatriculas"/>
   <addaction name="menuCalificaciones"/>
   <addaction name="menuProfesores"/>
   <addaction name="menuAyudas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFicha_alumno">
   <property name="text">
    <string>Ficha alumno</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionSalir_2">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionMatricular_alumno">
   <property name="text">
    <string>Matricular alumno</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionDesmatricular_alumno">
   <property name="text">
    <string>Desmatricular alumno</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionAprobar_alumno">
   <property name="text">
    <string>Aprobar alumno</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFicha_profesor">
   <property name="text">
    <string>Ficha profesor</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>d</string>
   </property>
  </action>
  <action name="actionAlta_alumno">
   <property name="text">
    <string>Alta alumno</string>
   </property>
  </action>
  <action name="actionAlta_alumno_2">
   <property name="text">
    <string>Alta alumno</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionMatricular_alumno_2">
   <property name="text">
    <string>Ficha alumno</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSalir_3">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
